cognite/powerops/preprocessor/utils.py

```diff
@@ -347,28 +360,49 @@ def group_files_by_metadata(
     return cut_files_by_group or {"default": files}
 
 
-def find_closest_file(files: list[FileMetadata], starttime_ms: float) -> Optional[FileMetadata]:
+def find_closest_file(files: list[FileMetadata], starttime_ms: float, sort_by: dict | None) -> Optional[FileMetadata]:
```

Comment:
I think would rename this to 
```suggestion
def find_earliest_file_after(files: list[FileMetadata], starttime_ms: float, sort_by: dict | None) -> Optional[FileMetadata]:
```
Then it is more clear what this function is doing.